app/Http/Controllers/WaitingRoomsController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\WaitingRoom;
+
+class WaitingRoomsController extends Controller
+{
+    private $validationRules = [
+        'campaign_id' => 'required|numeric',
+        'campaign_run_id' => 'required|numeric',
+        'signup_start_date' => 'required|date',
+        'signup_end_date' => 'required|date'
+    ];
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $rooms = WaitingRoom::all();
+
+        return view('waitingrooms.index')->withRooms($rooms);
```

Comment:
Just feels more consistent if it's all one method. The magic methods are nice, but feels like it takes an extra bit of mental effort especially if magic methods used for all different variables and how they're named.